packages/polymath-issuer/src/pages/compliance/components/AddApprovalForm.js

```diff
@@ -202,21 +202,22 @@ const formikEnhancer = withFormik({
   },
   handleSubmit: (values, { errors, setFieldError, props }) => {
     const { dispatch, handleClose, approvals } = props;
-    const approvalExists = approvals.find(
-      i =>
-        i.fromAddress === values.fromAddress && i.toAddress === values.toAddress
-    );
-    if (approvalExists) {
-      setFieldError(
-        'toAddress',
-        'An approval with both addresses already exists'
```

Comment:
Needs to be reworked, the approvals that expire keep being fetched. Unfortunately because of this the validations would prevent a user from being able to apply a new approval.

Reply:
Updated the validation logic @remon-nashid